.github/workflows/python-app.yml

```
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu-latest ]
-        python-version: [ 3.9 ]
+        python-version: [ 3.9, 3.10 ]
```

```suggestion
        python-version: [ 3.9, '3.10' ]
```
Or is this not required here?

---

It is, but I wanted to check with one job first.